src/webgpu/util/conversion.ts

```diff
@@ -272,10 +272,11 @@ export class VectorType {
   }
 }
 
-const vectorTypes = new Map<{ width: number; elementType: ScalarType }, VectorType>();
+// Maps a string representation of a vector type to vector type.
+const vectorTypes = new Map<string, VectorType>();
 
 export function TypeVec(width: number, elementType: ScalarType): VectorType {
-  const key = { width, elementType };
```

Comment:
This code doesn't work because any time you create a new dictionary like this, it's a new object that never compares identically-equal to a previously created on.

Reply:
TIL. Thanks for the fix!